openedx/core/djangoapps/content/block_structure/signals.py

```diff
@@ -15,10 +15,6 @@ def _listen_for_course_publish(sender, course_key, **kwargs):  # pylint: disable
     Catches the signal that a course has been published in the module
     store and creates/updates the corresponding cache entry.
     """
-    clear_course_from_cache(course_key)
```

Comment:
Why was this deleted?  We should synchronously clear the cache as soon as the signal is received - otherwise, we would be susceptible to data integrity issues.